dist/pathfora.js

```diff
@@ -916,6 +913,39 @@
         switch (config.layout) {
         case 'modal':
         case 'slideout':
+          // Add Content recommendation if we have one
```

Comment:
might be better to do this as its own function. i can see future usecases for including content recommendation in a gated widget or a future `inline` type. having it broken out makes it easier to integrate into other places? 

also easier to write tests against probably